assets/queries/dockerCompose/no_new_privileges_not_set/query.rego

```diff
@@ -0,0 +1,20 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	resource := input.document[i]
+	service_parameters := resource.services[name]
+    sec_opt := service_parameters.security_opt
+    sec_opt[s] != "no-new-privileges:true"
+    sec_opt[s] != "no-new-privileges"
```

Comment:
The policy is verifying each index of security_opt (and should not). I think we should look if there is no security_opt set to "no-new-privileges:true". 

```suggestion
    not no_new_privileges(sec_opt)
```

```
no_new_privileges(sec_opt) {
    sec_opt[_] == "no-new-privileges:true"
}
```